app/Http/Controllers/Api/V1/GoogleClassroomController.php

```diff
@@ -136,108 +139,119 @@ public function saveAccessToken(Request $request){
     /**
      * Copy whole project to google classroom
      */
-    public function copyProject(Request $request){
+    public function copyProject(Project $project, Request $request){
+        $allowed = $this->checkPermission($project);
+
+        if (!$allowed) {
+            return response([
+                'errors' => ['Forbidden. You are trying to share other user\'s project.'],
+            ], 403);
+        }
+
         try {
-            
-            $validator = Validator::make($request->all(), [
-                //'currentuser' => 'required',
-                'projectId' => 'required',
-            ]);
-    
-            if ($validator->fails()) {
-                return responseError([], 'Validation errors occured');
-            }
+           
             $authenticated_user = auth()->user();
```

Comment:
camelCase or _underscore? I see mix of both

Reply:
I had changed it to camel case.
$authUser